### PR TITLE
docs(timeline): update users guide

### PR DESCRIPTION
**Pull request by YuCJ**
the content will be published to Medium [報導者開放實驗室 The Reporter Lab](https://medium.com/twreporter) after the tool is released

### CODE REVIEW

File: packages/timeline/docs/zh-tw/使用說明.md [hunk 1]
```diff
@@ -1,37 +1,74 @@
-# @twreporter/timeline 使用說明
+# 「大事記圖表產生器」新聞小工具發佈！－－《報導者開放實驗室 The Reporter Lab》
```
Comment (nickhsine) [inline packages/timeline/docs/zh-tw/使用說明.md:1]: 「報導者／大事記」產生器？

File: packages/timeline/docs/zh-tw/使用說明.md [hunk 2]
```diff
@@ -1,37 +1,74 @@
...
 
-這是非營利獨立媒體[《報導者》](https://www.twreporter.org)製作的「大事記產生器」，可以幫你把時間序列資料轉換成網頁版大事記！
+這是非營利獨立媒體[《報導者》](https://www.twreporter.org)所製作的「大事記」元件產生器，可以幫你把時間序列資料，轉換成網頁版大事記圖表！
```
Comment (nickhsine) [inline packages/timeline/docs/zh-tw/使用說明.md:3]: 「大事記」-> 「報導者／大事記」

File: packages/timeline/docs/zh-tw/使用說明.md [hunk 3]
```diff
@@ -1,37 +1,74 @@
...
+請注意，在授權機器人帳號讀取時，要記得**將「通知邀請對象」打勾**
 
-[轉換器]() <!-- TODO: Sheet2Code Timeline 網址 -->
+![共用對話框（private）截圖](../assets/share-spreadsheet-private.png)
+
+如果沒有勾選寄信通知，機器人會無法得到權限：
+
+![共用對話框未寄信通知警告截圖](../assets/share-spreadsheet-warning-without-mail.png)
+
+### 3. 將你的資料表連結貼到大事記元件產生器，取得 embedded code
```
Comment (nickhsine) [inline packages/timeline/docs/zh-tw/使用說明.md:59]: 大事記 -> 「報導者／大事記」

File: packages/timeline/docs/zh-tw/使用說明.md [hunk 4]
```diff
@@ -1,37 +1,74 @@
...
+
+[大事記元件產生器]() <!-- TODO: Sheet2Code Timeline 網址 -->
```
Comment (nickhsine) [inline packages/timeline/docs/zh-tw/使用說明.md:61]: 「報導者／大事記」元件產生器？

File: packages/timeline/docs/zh-tw/使用說明.md [hunk 5]
```diff
@@ -79,6 +117,12 @@ element（元件）是組成一張大事記的最小單位，目前有三種 ele
...
 詳細授權資訊請見[授權條款頁](https://github.com/twreporter/orangutan-monorepo/blob/master/LICENSE)。
 
-## 回饋
+## 意見回饋
+
+我們將程式碼公開放置於 [Github](https://github.com/twreporter/orangutan-monorepo/tree/master/packages/timeline)，任何回饋或問題回報都歡迎直接在 Github 上開 Issue 或來信至 `developer@twreporter.org` 信箱，我們將有專人為你服務 :kissing_heart:。
+
+## 報導者開放實驗室
+
+「大事記圖表產生器」這是我們所製作的各種新聞小工具其中之一，若您對更多新聞小工具有興趣，歡迎追蹤我們的頁面：
```
Comment (nickhsine) [inline packages/timeline/docs/zh-tw/使用說明.md:126]: 「報導者／大事記」元件產生器？

我們好像要統一一下所有的名詞用法？

**Comment by YuCJ** [comment]
1. naming unified and wording updated
2. submitted to Medium as draft https://medium.com/twreporter/twreporter-lab-timeline-news-tool-4fe8db1035f2

**Comment by YuCJ** [comment]
The post has been updated by the editors at Medium:
https://medium.com/twreporter/twreporter-lab-timeline-news-tool-4fe8db1035f2

**Pull request merged**